rts/Lua/LuaHandle.cpp

```diff
@@ -3336,6 +3336,33 @@ string CLuaHandle::GetTooltip(int x, int y)
  * @bool internal
  */
 
+/*** Called when a command is issued.
+ *
+ * @function ActiveCommandChanged
+ * @int cmdID|nil
+ * @tparam table|nil cmdParams
+ * @tparam cmdOpts|nil options
+ */
+void CLuaHandle::ActiveCommandChanged(const SCommandDescription* cmdDesc)
+{
+	RECOIL_DETAILED_TRACY_ZONE;
+	LUA_CALL_IN_CHECK(L, false);
+	luaL_checkstack(L, 3, __func__);
```

Comment:
IIRC the checkstack value is 3+params (both in and out), but I have no idea why this is so.
https://github.com/beyond-all-reason/spring/blob/3d78bba9d8577f6684121861ce6a8f9c38a36c4f/rts/Lua/LuaHandle.cpp#L743

Anyway, in this case there can be 2 params so perhaps:
```suggestion
	luaL_checkstack(L, 5, __func__);
```